odl/datasets/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Access to real or simulated datasets for use with ODL."""
+
+from __future__ import absolute_import
+
+
+__all__ = ('ray_transform',)
+
+from . import ray_transform
```

Comment:
I'm not sure here. Are we not afraid of a growth in phantoms here? I suspect we could long term end up with a very large number of data sets.